src/components/Pill/style.css

```diff
@@ -0,0 +1,51 @@
+@import '../../shared/colors.css';
+
+.pill {
```

Comment:
I love CSS modules, but we've been leaning more heavily on `styled-system` and `styled-components` for new components. I may be outvoted here, but we should decide on using CSS modules or styled-components exclusively rather than mixing the two.

@pixelbandito @chelshaw Any thoughts?

Reply:
css modules are much easier tho. Haven't worked with styled-components.

Reply:
@andrewBalekha I definitely agree. In my experience with `styled-components` it has already proven to be tricky. We were hoping to take advantage of some features that `styled-system` and `styled-components` provided but it may be more overhead than it's worth.

@pixelbandito may have some more thoughts on this after a conversation I had with him yesterday afternoon. Anything to weigh in here?

Reply:
I'm trying to make styled components work with hover states in another branch. My take right now is that it's way too hard. 

I'm ok with going back to plain old css, but we should do a pros/cons in Slack or a GitHub issue and give other contributors and front end folks a chance to weigh in.

Reply:
After polling, and talking at the meeting, I think we should table the discussion and allow CSS modules to proliferate, so @andrewBalekha's code here is mostly fine.

However, we've been consistently PascalCasing CSS module class names when they refer to the main wrapper of a component and share the component's name. 

So it should be `.Pill {`